scripts/partition_manager.py

```diff
@@ -13,9 +13,12 @@
 
 
 def remove_item_not_in_list(list_to_remove_from, list_to_check):
+    to_remove = list()
```

Comment:
list_to_remove_from = [x for x in list_to_remove_from if x not in list_to_check and x is not 'app']

Reply:
Unfortunately this does not work, as changing the reference (I don't know the precise term) of the argument does not update the value at the caller side. 

```
def foo(bar):
  bar = [1,2]

def foo_too(bar):
  bar.append(1)

mylist = [0]

foo(mylist) # mylist = [0]

foo_too(mylist) # mylist = [0,1]
```

Reply:
Also, using `list_to_remove_from` to be iterated over re-introduces the original issue of iterating over the list being modified.

I found a fix for both of these issues:

`[list_to_remove_from.remove(x) for x in list_to_remove_from.copy() if x not in list_to_check and x is not 'app']`

Which seems to work fine, will update the PR :)